### DISCUSSION TITLE
Measure the distance as the crow flies

### DISCUSSION THREAD

**Discussion by Raiden-GH** [organicmaps/organicmaps#6040] [Ideas]
Hello,
I like the app and really enjoy using it.
In my opinion, Organic Maps would be even better if one could measure the distance as the crow flies (straight line) between two points on the map, like in Google Maps.

**Comment by hb0nd** [comment]
well in that case you'll be happy to know it'll be in the next release! 😄
https://github.com/organicmaps/organicmaps/pull/5381

**Reply by Raiden-GH** [reply]
🥳👍
